sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Common/DefaultAnalyzerConfiguration.cs

```diff
@@ -0,0 +1,44 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2018 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.IO;
+using System.Linq;
+using Microsoft.CodeAnalysis.Diagnostics;
+using SonarAnalyzer.Rules;
+
+namespace SonarAnalyzer.Common
+{
+    public class DefaultAnalyzerConfiguration : IAnalyzerConfiguration
+    {
+        public string GetProjectOutput(AnalyzerOptions options)
+        {
+            var projectOutputAdditionalFile = options.AdditionalFiles
+                .FirstOrDefault(UtilityAnalyzerBase.IsProjectOutput);
+
+            if (projectOutputAdditionalFile == null)
+            {
+                return null;
+            }
+
+            return File.ReadAllLines(projectOutputAdditionalFile.Path)
```

Comment:
I don't know if it's better to read the file directly ourselves, or use the _SourceText_ that Roslyn produces (?_SourceText.ToString()_?)

Reply:
Perhaps. I will create a ticket about that.

Reply:
@valhristov  can you please add a link to this ticket ?